src/scenes/home/benefit/benefit.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import Section from 'shared/components/section/section';
+import LinkButton from 'shared/components/linkButton/linkButton';
+import styles from './benefit.css';
+
+const Benefit = () => (
+  <div>
+    <Section title="Operation Code Veteran’s Day Benefit Dinner & Silent Auction" theme="white">
+      <div>
```

Comment:
When using the <Section> component, know that you can dive _right_ into the content (no need for a nested <div>). It actually breaks the styles for the component when you do so.